src/messaging/tests/echo/echo_requester.cpp

```diff
@@ -48,7 +48,7 @@
 namespace {
 
 // Max value for the number of EchoRequests sent.
-constexpr size_t kMaxEchoCount = 3;
+constexpr size_t kMaxEchoCount = 1;
```

Comment:
What's the motivation for this change?

Reply:
Good catch, this is from my test change, I should not include it into this PR.